extra/html-extra/Tests/Fixtures/cva.test

```diff
@@ -0,0 +1,37 @@
+--TEST--
+"cva" function
+--TEMPLATE--
+{% set alert = html_cva({
+    base: ['alert'],
+    variants: {
+       color: {
+           blue: 'alert-blue',
+           red: 'alert-red',
+           green: 'alert-green',
+           yellow: 'alert-yellow',
+       },
+       size: {
+           sm: 'alert-sm',
+           md: 'alert-md',
+           lg: 'alert-lg',
+       },
+       rounded: {
+           sm: 'rounded-sm',
+           md: 'rounded-md',
+           lg: 'rounded-lg',
+       }
+    },
+    compoundVariants: [{
+       color: ['red'],
+       size: ['lg'],
+       class: 'font-semibold'
+    }],
+    defaultVariants: {
+       rounded: 'md'
+    }
+}) %}
+{{ alert.apply({color: 'blue', size: 'sm'}) }}
+--DATA--
+return []
```

Comment:
Can you maybe add a test where you pass a CVA object directly to the template instead of defining it in the template (maybe not a great use case, but still)?